sickbeard/tvcache.py

```diff
@@ -38,62 +40,66 @@ def __init__(self, providerName):
         # Create the table if it's not already there
         try:
             if not self.hasTable(providerName):
-                logger.log(u"Creating cache table for provider {}".format(providerName), logger.DEBUG)
+                logger.log('Creating cache table for provider {0}'.format(providerName), logger.DEBUG)
                 self.action(
-                    "CREATE TABLE [" + providerName + "] (name TEXT, season NUMERIC, episodes TEXT, indexerid NUMERIC, url TEXT, time NUMERIC, quality NUMERIC, release_group TEXT)")
+                    b'CREATE TABLE [{provider_name}] (name TEXT, season NUMERIC, episodes TEXT, indexerid NUMERIC, '
+                    b'url TEXT, time NUMERIC, quality NUMERIC, release_group TEXT)'.format(provider_name=providerName))
             else:
-                sql_results = self.select("SELECT url, COUNT(url) AS count FROM [" + providerName + "] GROUP BY url HAVING count > 1")
+                sql_results = self.select(b'SELECT url, COUNT(url) AS count FROM [{provider_name}] '
+                                          b'GROUP BY url HAVING count > 1'.format(provider_name=providerName))
 
                 for cur_dupe in sql_results:
-                    self.action("DELETE FROM [" + providerName + "] WHERE url = ?", [cur_dupe["url"]])
+                    self.action(b'DELETE FROM [{provider_name}] WHERE url = ?'.format(provider_name=providerName), [cur_dupe[b'url']])
 
             # remove wrong old index
-            self.action("DROP INDEX IF EXISTS idx_url")
+            self.action(b'DROP INDEX IF EXISTS idx_url')
 
             # add unique index to prevent further dupes from happening if one does not exist
-            logger.log(u"Creating UNIQUE URL index for {}".format(providerName), logger.DEBUG)
-            self.action("CREATE UNIQUE INDEX IF NOT EXISTS idx_url_" + providerName + " ON [" + providerName + "] (url)")
+            logger.log(b'Creating UNIQUE URL index for {0}'.format(providerName), logger.DEBUG)
```

Comment:
You mean this one? Else I don't know where I need to remove the b''. The log is a mistake indeed.